nvm.sh

```diff
@@ -1725,9 +1725,22 @@ nvm_download_artifact() {
     TARBALL_URL="${MIRROR}/${SLUG}.tar.${COMPRESSION}"
   fi
 
-  if nvm_compare_checksum "${TARBALL}" "${CHECKSUM}" >/dev/null 2>&1; then
-    nvm_err "Checksums match! Using existing downloaded archive $(nvm_sanitize_path "${TARBALL}")"
-  else
+  local USE_LOCAL_TARBALL
+  if [ -r "${TARBALL}" ]; then
+    nvm_err "Local cache found:"
+    nvm_err "$(nvm_sanitize_path "${TARBALL}")"
```

Comment:
I wouldn't really be concerned with line length - long lines wrap, and the important part would be in the front.